canonical-web-and-design/canonical.com
# Add redirect for projects directory

## Done
Add redirect for projects directory

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/projects/directory
- Check it redirects to `#projects`

## Issue / Card
Fixes #149
